src/migrate.py

```diff
@@ -0,0 +1,49 @@
+from tv import app, db
+import os
+import re
+import importlib
+
+def ensure_migration_table():
+    with app.app_context():
+        table = db.session.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='migration'")
+        if len(table.fetchall()) != 1:
+            print("Creating migration table")
+            db.session.execute("CREATE TABLE migration (version INTEGER)")
+            db.session.execute("INSERT INTO migration (version) VALUES (0)")
```

Comment:
My SQL knowledge is pretty bad, but how is the variable version found here?

Reply:
what do you mean? its not a variable, its the name of a column in the migration table, and its adding a row to the table with a value of 0 in the versions column

Reply:
oh, sure, read the `!= 1` wrong and therefore got into the wrong context